sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -134,6 +135,16 @@ def _from_generated(cls, generated):
         )
 
 
+class ProtocolProperties(GeneratedProtocolSettings):
+    def __init__(self, smb_settings=None):
+        self.smb_settings = smb_settings
+
+    @classmethod
+    def _from_generated(cls, generated):
```

Comment:
where is this used?

Reply:
In the `service_properties_deserialize` (recent commit)